esp/services/ws_cloud/ws_cloudService.cpp

```diff
@@ -54,6 +54,9 @@ bool CWsCloudEx::onGetPODs(IEspContext& context, IEspGetPODsRequest& req, IEspGe
 {
     try
     {
+        if(context.getClientVersion() < 1.02)
```

Comment:
When no version is specified is this hit (I don't think it is, from memory)?

Reply:
Using curl with no version specified, and a browser when going to the 'web form' page both default to the newest version, 1.02